src/contactDesktop/Controller.java

```diff
@@ -0,0 +1,92 @@
+package contactDesktop;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+import jodd.json.JsonParser;
+import jodd.json.JsonSerializer;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+import java.util.Observable;
+import java.util.ResourceBundle;
+import java.util.Scanner;
+
+public class Controller implements Initializable {
+    File file = new File("contacts.json");
+    FileWriter fw = new FileWriter(file);
+    ObservableList<Contact> contacts = FXCollections.observableArrayList();
+
+    @FXML
+    ListView list;
+
+    @FXML
+    TextField nameText, phoneText, emailText;
+
+    public Controller() throws IOException {
+    }
+
+    public void addContact() throws IOException {
+        String name = nameText.getText();
+        String phoneNumber = phoneText.getText();
+        String email = emailText.getText();
+
+        if ((name.isEmpty() || phoneNumber.isEmpty() || email.isEmpty())){
```

Comment:
This would probably be better served by creating a `isValid` method inside of the `Contact` object.  Then the logic could stay localized.